shared-libs/transitions/test/unit/transitions/update_clinics.js

```diff
@@ -308,9 +308,51 @@ describe('update clinic', () => {
       assert(changed);
       assert(!doc.contact);
       assert.equal(doc.errors.length, 1);
-      assert.deepEqual(doc.errors[0], { code: 'sys.facility_not_found', message: 'translated' });
+      assert.deepEqual(doc.errors[0], {
+        code: 'sys.facility_not_found',
+        message: 'messages.generic.sys.facility_not_found'
```

Comment:
This test adds stubs the `utils.translate` function to generate a translated message. How come the message comes back not translated?

Reply:
Fixed.